### DISCUSSION TITLE
Send message with empty text

### DISCUSSION THREAD

**Discussion by levmorgot** [flyerhq/flutter_chat_ui#663] [Ideas & Requests]
Hello 👋 

How can I override _handleSendPressed so that I can send messages without text (when sharing a post, a picture)

 Thanks)

**Comment by demchenkoalex** [comment]
Hey! I think for attachments I did another button and did not plan to "send" button sending anything but provided text (since I do not support sending images with text yet), so not sure this will be possible right now :( but taking a note to fix in `v2`
